content/lxd/introduction.md

```diff
@@ -20,6 +20,12 @@ You can find a series of howtos and tutorials on YouTube:
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLddduKsl-KEhleT9VTR4hbtlNdtMr6cFd" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## Third-party integrations
```

Comment:
I think I'd move this section to be after the Features section.

Reply:
I moved it after `Availability`, so that the regular installation methods get displayed first.